### PR TITLE
Swap app and init filenames and references

### PR DESCRIPTION
**Pull request by keirog**

### CODE REVIEW

**Comment by laurieboyes** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   ],
   "scripts": {
     "test": "make test",
-    "start": "./node_modules/.bin/forever server/app.js"
+    "start": "./node_modules/.bin/forever server/init.js"
```
Comment (matthew-andrews) [inline package.json]: `./node_modules/.bin` shouldn't be needed inside `package.json` scrypts

File: Dockerfile [hunk 2]
```diff
@@ -1 +0,0 @@
-FROM financialtimes/next-heroku:0.12.6
```
Comment (matthew-andrews) [inline Dockerfile]: lol wow

File: package.json (hunk 1 above)
Comment (keirog) [inline package.json]: I suppose we shouldn't encourage using `npm start` anyway. Will remove

Comment (matthew-andrews) [inline package.json]: Haikro will happily fall back to using npm start if no Procfile.

Also you shouldn't need `./node_modules/.bin` inside the Procfile either

Comment (keirog) [inline package.json]: Ok ta

**Pull request merged**